package.json

```diff
@@ -50,21 +50,21 @@
     "remap-istanbul": "^0.9.5",
     "sinon": "^2.3.6",
     "ts-node": "^3.3.0",
-    "tslint": "^5.6.0",
+    "tslint": "^5.7.0",
     "typed-css-modules": "^0.3.0",
     "typescript": "~2.4.2",
-    "webpack": "^3.5.4"
+    "webpack": "^3.7.1"
   },
   "dependencies": {
-    "@dojo/core": "next",
-    "@dojo/has": "next",
-    "@dojo/loader": "next",
-    "@dojo/routing": "v2.0.0-beta2.1",
-    "@dojo/shim": "next",
-    "@dojo/widget-core": "next",
-    "@dojo/widgets": "2.0.0-alpha.24",
-    "maquette": "^2.5.3",
-    "monaco-editor": "^0.9.0",
-    "source-map": "^0.5.6"
+    "@dojo/core": "0.1.0",
+    "@dojo/has": "0.1.0",
+    "@dojo/loader": "0.1.0",
+    "@dojo/routing": "0.1.0",
+    "@dojo/shim": "0.1.0",
+    "@dojo/widget-core": "0.1.0",
```

Comment:
you'll want this to be `0.1.1` as we had to do a patch... but in general you'll be able to tilde the @dojo deps if you want.